test/expected/cluster.out

```diff
@@ -0,0 +1,94 @@
+CREATE TABLE cluster_test(time timestamptz, temp float, location int);
+SELECT create_hypertable('cluster_test', 'time', chunk_time_interval => interval '1 day');
+NOTICE:  Adding NOT NULL constraint to time column time (NULL time values not allowed)
+ create_hypertable 
+-------------------
+ 
+(1 row)
+
+-- Show default indexes
+SELECT * FROM test.show_indexes('cluster_test');
+         Index         | Columns | Unique | Primary | Exclusion | Tablespace 
+-----------------------+---------+--------+---------+-----------+------------
+ cluster_test_time_idx | {time}  | f      | f       | f         | 
+(1 row)
+
+-- Create two chunks
+INSERT INTO cluster_test VALUES ('2017-01-20T09:00:01', 23.4, 1),
+       ('2017-01-21T09:00:01', 21.3, 2);
+-- Run cluster
+CLUSTER VERBOSE cluster_test USING cluster_test_time_idx;
+INFO:  clustering "_timescaledb_internal._hyper_1_2_chunk" using index scan on "_hyper_1_2_chunk_cluster_test_time_idx"
+INFO:  "_hyper_1_2_chunk": found 0 removable, 1 nonremovable row versions in 1 pages
+INFO:  clustering "_timescaledb_internal._hyper_1_1_chunk" using index scan on "_hyper_1_1_chunk_cluster_test_time_idx"
+INFO:  "_hyper_1_1_chunk": found 0 removable, 1 nonremovable row versions in 1 pages
+INFO:  clustering "public.cluster_test" using sequential scan and sort
+INFO:  "cluster_test": found 0 removable, 0 nonremovable row versions in 0 pages
+-- Create a third chunk
+INSERT INTO cluster_test VALUES ('2017-01-22T09:00:01', 19.5, 3);
+-- Show clustered indexes
+SELECT indexrelid::regclass, indisclustered
+FROM pg_index
+WHERE indisclustered = true;
+                          indexrelid                          | indisclustered 
+--------------------------------------------------------------+----------------
+ _timescaledb_internal._hyper_1_2_chunk_cluster_test_time_idx | t
```

Comment:
I would find it more straightforward to set the clustered index on new chunks before doing a news cluster on the hypertable. Namely, if you now `CLUSTER _timescaledb_internal._hyper_1_3_chunk` I think this would fail. That's a bit weird.